apollo-test/src/main/java/com/spotify/apollo/test/StubClient.java

```diff
@@ -193,7 +204,12 @@ public void clear() {
    * Configure a constant (i.e., all matching requests will always result in the same response)
    * response for some request. The returned builder allows configuration of payload (default
    * no payload) and delay before the response is sent (default 0).
+   *
+   * When using this method to define request to response mappings, the mappings will be evaluated
+   * in the order which they were added.
+   * @deprecated in favour of {@link #when(Matcher)} or {@link #when(String)}.
    */
+  @Deprecated
```

Comment:
It's because I don't like solutions that allow you to do the same thing in two different ways. I prefer being opinionated about how things should work, it makes things simpler. So I would like to recommend people to use only the new version of the API.